### PR TITLE
Add lvmd.yaml and ovn.yaml default configuration files

### PR DESCRIPTION
**Pull request by ggiguash**
Added lvmd.yaml.default and ovn.yaml.default configuration files following the format in the existing config.yaml.default file

Closes:
- [USHIFT-822](https://issues.redhat.com//browse/USHIFT-822)
- [USHIFT-923](https://issues.redhat.com//browse/USHIFT-923)

### CODE REVIEW

File: packaging/microshift/config.yaml [hunk 1]
```diff
@@ -1,31 +1,18 @@
...
-  #nodeIP: ''
-
-apiServer:
-  # The Subject Alternative Names for the external certificates in API server (defaults to hostname -A)
-  #subjectAltNames: []
-
-debugging:
-  # Log verbosity ('Normal', 'Debug', 'Trace', 'TraceAll'):
-  #logLevel: 'Normal'
+# See https://github.com/openshift/microshift/blob/main/docs/howto_config.md#configuration
```
Comment (dhellmann) [inline packaging/microshift/config.yaml:1]: I'm worried that if we link to GitHub in the example files then we have to maintain those URLs for the product lifetime. Can we either link to product docs, or drop the links entirely and assume that users will search for the docs themselves?

File: packaging/microshift/lvmd.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# See https://github.com/openshift/topolvm/blob/main/docs/lvmd.md
+# for more information on the configuration options and their values.
+#
+# socket-name: /run/topolvm/lvmd.sock
```
Comment (dhellmann) [inline packaging/microshift/lvmd.yaml:4]: We should  make this example match the defaults in the code.

**Review by dhellmann — CHANGES_REQUESTED** [review]

File: packaging/microshift/ovn.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# See https://github.com/openshift/microshift/blob/main/docs/network/default_cni_plugin.md#configuring-ovn-kubernetes
+# for more information on the configuration options and their values
+#
+# ovsInit:
+#   disableOVSInit: true
+#   gatewayInterface: eth0
+#   externalGatewayInterface: eth1
```
Comment (zshi-redhat) [inline packaging/microshift/ovn.yaml:7]:
```suggestion
#   disableOVSInit: false
#   gatewayInterface: ""
#   externalGatewayInterface: ""
```

**Comment by ggiguash** [comment]
@zshi-redhat / @dhellmann, I updated the config files to follow the format of config.yaml
Is this better now?

File: packaging/microshift/lvmd.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    #default: true
+
+    # The number of stripes in the logical volume
+    #stripe: ""
+
+    # The amount of data that is written to one device before moving to the next device
+    #stripe-size: ""
+
+    # Extra arguments to pass to lvcreate, e.g. ["--type=raid1"]
+    lvcreate-options:
```
Comment (dhellmann) [inline packaging/microshift/lvmd.yaml:24]: Should we just go ahead and comment this out, too? Or maybe make it a valid empty list by adding `[]` to the end of the line?

Comment (ggiguash) [reply packaging/microshift/lvmd.yaml:24]: I'll comment it out.

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Pull request merged**